hawc/apps/assessment/serializers.py

```diff
@@ -192,3 +192,15 @@ class DatasetSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Dataset
         fields = "__all__"
+
+
+class JobSerializer(serializers.ModelSerializer):
```

Comment:
should I allow string representations of jobs in POST requests? `{"job":"TEST"}` and `{"job":1}` are both valid?

Reply:
does this work? https://stackoverflow.com/a/28954424

I'm fine with whatever the default behavior is; we can make it easier to use in the client with an enum if we need to